src/containers/Tenant/Diagnostics/HotKeys/HotKeys.tsx

```diff
@@ -0,0 +1,143 @@
+import type {IResponseError} from '../../../../types/api/error';
+import {Icon} from '../../../../components/Icon';
+import {ResponseError} from '../../../../components/Errors/ResponseError';
+import {useTypedSelector} from '../../../../utils/hooks';
+import {cn} from '../../../../utils/cn';
+import {DEFAULT_TABLE_SETTINGS} from '../../../../utils/constants';
+import {
+    setHotKeysData,
+    setHotKeysDataWasNotLoaded,
+    setHotKeysError,
+    setHotKeysLoading,
+} from '../../../../store/reducers/hotKeys/hotKeys';
+
+import './HotKeys.scss';
+import i18n from './i18n';
+
+const b = cn('ydb-hot-keys');
+
+const tableColumnsIds = {
+    accessSample: 'accessSample',
+    keyValues: 'keyValues',
+} as const;
+
+const getHotKeysColumns = (keyColumnsIds: string[] = []): Column<HotKey>[] => {
+    const keysColumns: Column<HotKey>[] = keyColumnsIds.map((col, index) => ({
+        name: col,
+        header: (
+            <div className={b('primary-key-column')}>
+                <Icon name="key" viewBox="0 0 12 7" width={12} height={7} />
+                {col}
+            </div>
+        ),
+        render: ({row}) => row.keyValues[index],
+        align: DataTable.RIGHT,
+        sortable: false,
+    }));
+
+    return [
+        {
+            name: tableColumnsIds.accessSample,
+            header: 'Samples',
+            render: ({row}) => row.accessSample,
+            align: DataTable.RIGHT,
+            sortable: false,
+        },
+        ...keysColumns,
+    ];
+};
+
+interface HotKeysProps {
+    path: string;
+}
+
+export function HotKeys({path}: HotKeysProps) {
+    const dispatch = useDispatch();
+
+    const collectSamplesTimerRef = useRef<ReturnType<typeof setTimeout>>();
+
+    const {loading, wasLoaded, data, error} = useTypedSelector((state) => state.hotKeys);
+    const {loading: schemaLoading, data: schemaData} = useTypedSelector((state) => state.schema);
+
+    const keyColumnsIds = schemaData[path]?.PathDescription?.Table?.KeyColumnNames;
+
+    const tableColumns = useMemo(() => {
+        return getHotKeysColumns(keyColumnsIds);
+    }, [keyColumnsIds]);
+
+    useEffect(() => {
+        const fetchHotkeys = async (enableSampling: boolean) => {
+            // Set hotkeys error, but not data, since data is set conditionally
+            try {
+                const response = await window.api.getHotKeys(path, enableSampling);
+                return response;
+            } catch (err) {
+                dispatch(setHotKeysError(err as IResponseError));
+                return undefined;
+            }
+        };
+
+        const fetchData = async () => {
+            // If there is previous pending request for samples, cancel it
+            if (collectSamplesTimerRef.current !== undefined) {
+                window.clearInterval(collectSamplesTimerRef.current);
+            }
+
+            dispatch(setHotKeysDataWasNotLoaded());
+            dispatch(setHotKeysLoading());
```

Comment:
Although there is no autorefresh for hotkeys tab, I made separate actions to keep flow similar to other components